src/04-modality-specific-files/09-positron-emission-tomography.md

```diff
@@ -0,0 +1,443 @@
+	pet/
+	sub-<participant_label>[_ses-<session_label>][_task-<task_label>][_acq-<label>][_rec-<label>][_run-<index>]_recording-<label>_blood.tsv
+	sub-<participant_label>[_ses-<session_label>][_task-<task_label>][_acq-<label>][_rec-<label>][_run-<index>]_recording-<label>_blood.json
+
+```
+
+Blood data belongs in the /pet folder along with the corresponding PET data.
+However, the blood data also follows the [inheritance principle](../02-common-principles.md#the-inheritance-principle) and may be moved to an upper level folder
+if it does not change, for example, with multiple reconstructions.
+The blood data is most often recorded using an autosampler for continuous blood samples,
+or manually for discrete blood samples.
+Therefore, the recording key (`recording-<label>`) has two reserved values:
+`autosampler`,
+for continuous blood data measurements;
+and `manual`, for discrete blood data.
+All blood measurements should be reported according to a single time-scale
+in relation to time zero defined by the PET data (Figure 1).
+All definitions used below are in accordance with Innis et al. 2007 ([doi:10.1038/sj.jcbfm.9600493](https://doi.org/10.1038/sj.jcbfm.9600493)).
+
+## Blood sidecar JSON (`*recording-(manual|autosampler)_blood.json`)
+
+If blood measurements were made a `*recording-<label>_blood.json` file is expected that contains
+information about which blood data (plasma, whole blood, and metabolites) is available as well as some blood measurement details.
+These are detailed in the next sections. Different fields are customary depending on if manual or automatic blood sampling is carried out.
+
+| **Key name**                        | **Requirement level** | **Data type** | **Description**                                                                                                                                                                                                                                               |
+| ----------------------------------- | --------------------- | ------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| PlasmaAvail                         | REQUIRED              | [boolean][]   | Boolean that specifies if plasma measurements are available. If this is false, all of the plasma fields should be excluded.                                                                                                                                   |
+| PlasmaFreeFraction                  | RECOMMENDED           | [number][]    | Measured free fraction in plasma, meaning the concentration of free compound in plasma divided by total concentration of compound in plasma.                                                                                                                  |
```

Comment:
what are the units? Is it expressed as a fraction, like 0.10, or as a percentage like 10% (which would numerically be 10.0)?

Reply:
```suggestion
| PlasmaFreeFraction                  | RECOMMENDED           | [number][]    | Measured free fraction in plasma, meaning the concentration of free compound in plasma divided by total concentration of compound in plasma (Units: 0-100%).                                                                                                                  |
```